katana/wui/core/templates/core/index_scripts.html

```diff
@@ -0,0 +1,72 @@
+{% load staticfiles %}
+
```

Comment:
Hi @satyakrish,

Can you please adjust the indentations of this file? Thanks.

Reply:
done